plugins/repro/src/App.tsx

```diff
@@ -0,0 +1,89 @@
+import { framer, ManagedCollection } from "framer-plugin"
+
+import { useState } from "react"
+import "./App.css"
+import { simpleHash, slugify } from "./utils"
+
+const contentFieldId = "content"
+
+const html = `
+<html>
+<p>example md file</p>
+<pre><code>import { z } from &#39;zod&#39;
+
+</code></pre>
+
+</html>
+`
+export async function importData(collection: ManagedCollection) {
+    try {
+        await collection.setFields([
+            {
+                type: "formattedText",
+                name: "Content",
+                id: contentFieldId,
+            },
+        ])
+
+        const unseenItemIds = new Set(await collection.getItemIds())
+
+        await collection.addItems([
```

Comment:
The error happens here

`Failed to upload images in formatted text for field: Content, in item: 00000078`